### DISCUSSION TITLE
Figma v5大概什么时候会发布？

### DISCUSSION THREAD

**Discussion by jaybowang** [ant-design/ant-design#39271] [Q&A] [answered]
团队一直在用Figma，正好遇到目前业务重构，再考虑要不要用v5。
但缺少Figma的支持会让人很纠结，v5视觉和交互升级感觉都挺不错的，希望能尽快用上🙏。

在升级到v5的讨论中有人提到过这个问题，单独提出来希望能看到。

**Comment by arvinxx** [comment, marked as answer]
官方不提供 V5 Figma 组件库。可以去问下Figma 社区的 antd 文件什么时候升级成 v5 吧。
